bundles/com.espressif.idf.launch.serial.ui/src/com/espressif/idf/launch/serial/ui/internal/CMakeMainTab2.java

```diff
@@ -792,6 +794,10 @@ private void updateArgumentsField() {
 		jtagArgumentsField.setText(configOptiopns);
 	}
 
+	private String newVariableExpression(OpenocdDynamicVariable dynamicVariable) {
+		return newVariableExpression(dynamicVariable.getValue(), null);
+	}
```

Comment:
I was thinking about moving this to OpenocdDynamicVariable responsibility, but I dropped this idea as there is little benefit as it is only used in this class. Also, this class already doing similar things with other variables.
In the future, we will probably add a similar thing to the TabMain class